### PR TITLE
chore: update CHANGELOG for 202212 release

### PR DESCRIPTION
**Pull request by scotthart**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -116,7 +116,12 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.5.0 - TBD
+## v2.6.0 - TBD
```
Comment (coryan) [inline CHANGELOG.md:119]: nit: we are doing this in the PR bumping version numbers.

Comment (devbww) [inline CHANGELOG.md:119]: I believe we decided that we wouldn't add this until the "bump" PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -116,7 +116,12 @@ may elicit a deprecation warning. See [#9086] for more details.
...
+
+## v2.5.0 - 2022-12
+
+**BREAKING CHANGES**
```
Comment (coryan) [inline CHANGELOG.md:123]: Hmmm... I would not say this is breaking.  Those were experimental APIs, subject to change without notice.  Maybe say:

```md
**NOTE**
- feat!: the functions returning bidirectional streams no longer have experimental markers
```

Thoughts?

File: CHANGELOG.md [hunk 3]
```diff
@@ -116,7 +116,12 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.5.0 - TBD
+## v2.6.0 - TBD
+
+## v2.5.0 - 2022-12
```
Comment (devbww) [inline CHANGELOG.md:121]: Remember that the "update CHANGELOG" PR should now also include an edit to CMakeLists.txt to set PROJECT_VERSION_PRE_RELEASE to the empty string.

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344)* status: 0 of 3 files reviewed, 3 unresolved discussions (waiting on @coryan and @devbww)

___
*[`CHANGELOG.md` line 119 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344#-NIE5i3f2fvNF8l-K5mo-r1-119:-NIE7jhI83DFuwH9slqK:bss4a2b) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/329bab6261d076894c6cd4b59efcb51b423a47f9/CHANGELOG.md#L119)):*
<details><summary><i>Previously, devbww (Bradley White) wrote…</i></summary><blockquote>

I believe we decided that we wouldn't add this until the "bump" PR.
</blockquote></details>

Removed.
___
*[`CHANGELOG.md` line 121 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344#-NIE5i3f2fvNF8l-K5mo-r1-121:-NIE7n-5Bj_gx4WOekuv:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/329bab6261d076894c6cd4b59efcb51b423a47f9/CHANGELOG.md#L121)):*
<details><summary><i>Previously, devbww (Bradley White) wrote…</i></summary><blockquote>

Remember that the "update CHANGELOG" PR should now also include an edit to CMakeLists.txt to set PROJECT_VERSION_PRE_RELEASE to the empty string.
</blockquote></details>

Done.
___
*[`CHANGELOG.md` line 123 at r1](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344#-NIE5i3f2fvNF8l-K5mo-r1-123:-NIE7nPe7grICmb1HjRF:bb73djm) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/329bab6261d076894c6cd4b59efcb51b423a47f9/CHANGELOG.md#L123)):*
<details><summary><i>Previously, coryan (Carlos O'Ryan) wrote…</i></summary><blockquote>

Hmmm... I would not say this is breaking.  Those were experimental APIs, subject to change without notice.  Maybe say:

```md
**NOTE**
- feat!: the functions returning bidirectional streams no longer have experimental markers
```

Thoughts?
</blockquote></details>

SGTM

File: CHANGELOG.md [hunk 4]
```diff
@@ -116,7 +116,11 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.5.0 - TBD
+## v2.5.0 - 2022-12
+
+**NOTE**
+
+- feat!: bidirectional streams are GA ([#10340](https://github.com/googleapis/google-cloud-cpp/pull/10340))
```
Comment (dbolduc) [inline CHANGELOG.md:123]: can we say:

```
We have dropped the experimental marker from bidirectional streaming APIs. The APIs in question are:
- `bigquery::BigQueryWriteClient::AsyncAppendRows()`
- `dialogflow_cx::SessionsClient::AsyncStreamingDetectIntent()`
- `dialogflow_es::ParticipantsClient::AsyncStreamingAnalyzeContent()`
- `dialogflow_es::SessionsClient::AsyncStreamingDetectIntent()`
- `logging::LoggingServiceV2Client::AsyncTailLogEntries()`
- `speech::SpeechClient::AsyncStreamingRecognize()`

If you use any of these APIs, you must drop the `ExperimentalTag` in your code, accordingly.
```

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344)* status: 0 of 3 files reviewed, 4 unresolved discussions (waiting on @coryan, @dbolduc, and @devbww)

___
*[`CHANGELOG.md` line 123 at r3](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344#-NIE5i3f2fvNF8l-K5mo-r3-123:-NIEBVRTEgmbZ-TBDWZr:b-896fix) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/2f3696ea3d989269c3ff5e6bcca7679692693246/CHANGELOG.md#L123)):*
<details><summary><i>Previously, dbolduc (Darren Bolduc) wrote…</i></summary><blockquote>

can we say:

```
We have dropped the experimental marker from bidirectional streaming APIs. The APIs in question are:
- `bigquery::BigQueryWriteClient::AsyncAppendRows()`
- `dialogflow_cx::SessionsClient::AsyncStreamingDetectIntent()`
- `dialogflow_es::ParticipantsClient::AsyncStreamingAnalyzeContent()`
- `dialogflow_es::SessionsClient::AsyncStreamingDetectIntent()`
- `logging::LoggingServiceV2Client::AsyncTailLogEntries()`
- `speech::SpeechClient::AsyncStreamingRecognize()`

If you use any of these APIs, you must drop the `ExperimentalTag` in your code, accordingly. See [#10340](https://github.com/googleapis/google-cloud-cpp/pull/10340) for more details.
```
</blockquote></details>

Done.

**Review by dbolduc — APPROVED** [review]

**Review by coryan — APPROVED** [review]
> Reviewed 2 of 3 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344)* status: all files reviewed, 2 unresolved discussions (waiting on @dbolduc and @devbww)

**Review by devbww — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -116,7 +116,19 @@ may elicit a deprecation warning. See [#9086] for more details.
 
 </details>
 
-## v2.5.0 - TBD
+## v2.5.0 - 2022-12
+
+**NOTE**
+
+- feat!: We have dropped the experimental marker from bidirectional streaming APIs ([#10340](https://github.com/googleapis/google-cloud-cpp/pull/10340)). The APIs in question are:
+    - `bigquery::BigQueryWriteClient::AsyncAppendRows()`
```
Comment (devbww) [inline CHANGELOG.md:124]: It looks like you wanted a sub-list there.

**Review by scotthart — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344)* status: 2 of 3 files reviewed, 3 unresolved discussions (waiting on @coryan, @dbolduc, and @devbww)

___
*[`CHANGELOG.md` line 124 at r4](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10344#-NIE5i3f2fvNF8l-K5mo-r4-124:-NIEDb39AU8-r2wvmKWq:b-zgkwtm) ([raw file](https://github.com/googleapis/google-cloud-cpp/blob/8906079a2c66a459a3d202ef17b35cf44b76c58c/CHANGELOG.md#L124)):*
<details><summary><i>Previously, devbww (Bradley White) wrote…</i></summary><blockquote>

It looks like you wanted a sub-list there.
</blockquote></details>

It renders as a sub list with either 2 or 4 spaces. Changed to 2 to satisfy checkers build.

**Pull request merged**